src/containers/LoginContainer.jsx

```diff
@@ -13,25 +14,43 @@ import { useNavigate } from 'react-router-dom';
 import Loading from '../components/Loading';
 import { useDispatch } from 'react-redux';
 import { getToken } from '../states/features/login/actions';
+import { API_URL } from '../constants';
+import InputPopup from '../components/InputPopup';
 
 const LoginContainer = () => {
-  const [loading, setLoading] = useState(false);
-  const [incorrectCred, setIncorrectCred] = useState(null);
-  const form = useForm();
-  const navigate = useNavigate();
-  const dispatch = useDispatch();
-  const { register, handleSubmit, formState } = form;
-  const { errors } = formState;
-  const mySubmit = async (data) => {
+const [loading, setLoading] = useState(false);
+const [incorrectCred, setIncorrectCred] = useState(null);
+const form = useForm();
+const navigate = useNavigate();
+const dispatch = useDispatch();
+const { register, handleSubmit, setValue, formState } = form;
+const { errors } = formState;
+
+const mySubmit = async (data) => {
+// remember the inputed data and save them in local storage
+if (data.rememberMe) {
+localStorage.setItem('rememberedEmail', data.email);
+localStorage.setItem('rememberedPassword', data.password);
```

Comment:
I would not suggest to store a user's password in localStorage without *hashing* it as it makes it vulnerable. Please hashing it using byrcyptjs (bycrypt for the frontend), you may refer to its documentation here: [bcrypt.js - npm](https://www.npmjs.com/package/bcryptjs), then decrypt it again on line 69 when you fetch it back from localStorage.